Source/Workers/decodeDraco.js

```diff
@@ -125,6 +127,7 @@ define([
 
     function decodeDracoPrimitive(parameters) {
         if (!defined(dracoDecoder)) {
+            draco = self.wasmModule;
```

Comment:
Or is this standard procedure for web worker tasks?

Reply:
This isn't necessarily standard. @shunter Do you know if there was a more straightforward way of passing this module to the web worker functions without altering their function signatures (too much).

Reply:
Right now it's assumed all web worker function are synchronous, and compiling the module is always asynchronous. I could restructure `createTaskProcessorWorker` to wait for a promise to be fulfilled if one is returned. 

But additionally, which module to require is based on whether or not Web Assembly is supported, as we pass the backup js module if it is not. That's why I have this separate from the decodeDraco worker itself, as I don't want to require both modules.

Reply:
I see now that you're actually loading the wasm binary on the main thread and transferring the data over to the worker.

I guess in general, my suggestion is that the protocol between the the main thread (`DracoLoader`) and the worker (`decodeDraco`) can be whatever you want.

Our existing workers have sort of a two-stage loading process:

1. The Worker is instantiated with `cesiumWorkerBootstrapper`, a special Worker which only understands bootstrap messages.
2. `TaskProcessor` posts a bootstrap message describing what worker module to transmogrify into.  `cesiumWorkerBootstrapper` loads that module and replaces `onmessage` in order to hand off control.

At this point all of our existing workers are fully-formed (I think) but that doesn't necessarily have to be the case.  You could add additional loading steps of your own design.  `decodeDraco` could expect its first message to contain the wasm binary or the path to the fallback script, which it could then load synchronously (which is preferable in a worker).  Then, it could switch modes and expect subsequent messages to be traditional task messages and process as usual.

On the main thread, `DracoLoader` could handle this protocol transparently by creating the `TaskProcessor` and then immediately scheduling the first message to finish the load.  

Ultimately you can do what you want, but the point of all this would be to tease apart these two concepts rather than fusing them together such that they have to know about each other.